core/src/main/java/org/mule/runtime/core/internal/exception/OnErrorPropagateHandler.java

```diff
@@ -7,15 +7,21 @@
 package org.mule.runtime.core.internal.exception;
 
 import static reactor.core.publisher.Mono.just;
+
+import org.mule.runtime.api.component.location.ComponentLocation;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.mule.runtime.api.component.location.ComponentLocation'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7640&issues=AWirRZ1EotIXnmsUDiqR&open=AWirRZ1EotIXnmsUDiqR)

[ISSUE_KEY]: AWirRZ1EotIXnmsUDiqR